src/Widgets/GameArea.vala

```diff
@@ -54,8 +54,11 @@ public class Warble.Widgets.GameArea : Gtk.Grid {
         initialize ();
     }
 
-    public void new_game () {
-        if (!can_safely_start_new_game ()) {
```

Comment:
This check was duplicate and handled by the caller, where the user was prompted if necessary